src/devtools/client/debugger/src/utils/ast.js

```diff
@@ -2,31 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-//
-
-export function findBestMatchExpression(symbols, tokenPos) {
```

Comment:
Looks like we still use `findBestMatchExpression` in preview.js:

https://github.com/RecordReplay/devtools/blob/master/src/devtools/client/debugger/src/actions/preview.js#L8